FireBrick/UI/AddTechnologyViewController/AddTechnologyViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  AddTechnologyViewController.swift
+//  FireBrick
+//
+//  Created by metoSimka on 27/05/2019.
+//  Copyright © 2019 metoSimka. All rights reserved.
+//
+
+import UIKit
+import SwiftEntryKit
+import Firebase
+import FirebaseFirestore
+
+class AddTechnologyViewController: UIViewController {
+    
+    var docRef: DocumentReference!
+    var db: Firestore?
+    
+    @IBOutlet weak var addButton: UIButton!
+    @IBOutlet weak var nameTextField: UITextField!
+    @IBOutlet weak var docTextField: UITextField!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        db = Firestore.firestore()
+        nameTextField.delegate = self
+        docTextField.delegate = self
+        updataButtonState()
+    }
+    
+    @IBAction func cancel(_ sender: UIButton) {
+        SwiftEntryKit.dismiss()
+    }
+    
+    @IBAction func add(_ sender: UIButton) {
+        writeData()
+    }
+    
+    func writeData() {
+        guard let name = nameTextField.text else {
+            return
+        }
+        guard let doc = docTextField.text else {
+            return
+        }
+        guard name.count > 0 && doc.count > 0 else {
+            return
+        }
+        
+        let dataToSave: [String: Any] = ["Name": name, "Documentation": doc]
```

Comment:
```suggestion
        let dataToSave: [String: Any] = ["name": name, "documentation": doc]
```
"Name" and "Documentation" keys for fields so they should be lowercase
also "Name" and "Documentation" should be extracted into constants